app/Http/Controllers/CompetitionsController.php

```diff
@@ -115,4 +116,18 @@ public function removeUser(Competition $competition, User $user)
 
         return redirect()->back()->with('status', 'User was removed from competition ' . $competition->id);
     }
+
+    /**
+     * Detach a user from a competition.
```

Comment:
it's showing and sending messages in a competition. I originally had in in the message controller, but needed more info from the competition, happy to move it back if that makes more sense though